examples/OneDShallowWaterGeostrophicAdjustment.jl

```diff
@@ -78,12 +78,13 @@ nothing #hide
 # and then creates `Vars` struct.
 """
     Vars(grid)
-Constructs Vars for 1D shallow water based on the dimensions of arrays of the `grid`.
+
+Construct Vars for 1D shallow water based on the dimensions of arrays of the `grid`.
```

Comment:
Should Vars here be in backticks to match line 78 above?
>and then creates `Vars` struct.

Could more info than 1D shallow water be given here? For the `calcN!` function in the docs we have
>Compute the nonlinear terms for 1D linear shallow water dynamics.

So maybe for the `Vars` constructor we could have something similar, e.g.

Construct the `Vars` for 1D linear shallow water dynamics based on the dimensions of the `grid` arrays.

Reply:
The equations are written in the liberated version of the example in the docs
https://fourierflows.github.io/FourierFlowsDocumentation/dev/literated/OneDShallowWaterGeostrophicAdjustment/

Since these Params/Vars/etc here are never documented I'm a bit hesitant to include the equations in the docstrings also.

Perhaps a better strategy would be to restructure this example a bit so that we create a module `OneDLinearShallowWater`.

Reply:
>Since these Params/Vars/etc here are never documented I'm a bit hesitant to include the equations in the docstrings also.

Definitely agree.

>Perhaps a better strategy would be to restructure this example a bit so that we create a module OneDLinearShallowWater.

That is a cool idea (and follows the suggestion from the docs) but I think as it is it is clear. In the context of the docs I think what was there originally for the `Vars` constructor was clear enough, I just thought that changing "1D shallow water" to "1D linear shallow water dynamics" reminded the user exactly what the `Vars` are for. Probably just me being overly pedantic!